testing-framework/ec2/base/scripts/create_domain_node_compose_file.sh

```diff
@@ -13,7 +13,7 @@ services:
     build:
       context: .
       dockerfile: $HOME/subspace/subspace/Dockerfile-node
-    image: \${NODE_ORG}/node:\${NODE_TAG}
+    image: \${NODE_ORG}/\${NODE_TAG}:latest
```

Comment:
this seems incorrect. `NODE_TAG` represents the docker tag for the node and why default to `latest` ?